### ISSUE TITLE
<Img/>  tags are overflowing trending repositories description

### ISSUE THREAD

**Issue by bumbeishvili** [Nightonke/GithubWidget#2]
Many repositories description starts with emoji (see screenshot bellow)
___
<img width="374" alt="screen shot 2016-05-06 at 10 50 05 am" src="https://cloud.githubusercontent.com/assets/6873202/15066200/624b8b44-1378-11e6-9b42-4f9110279b6e.png">

___

 It will be good if you cut this `img` tags from repository description text, because, it will be possible to read some description of repository, without opening in new tab

**Comment by Nightonke** [comment]
Yep, I've found this problem too. I will make a optimization as soon as possible.
